recipes-core/images/includes/nilrt-proprietary.inc

```diff
@@ -9,6 +9,7 @@ LDCONFIGDEPEND = ""
 
 NI_PROPRIETARY_SAFEMODE_PACKAGES = "\
         ni-software-installation-daemon \
+        ni-wireless-cert-management-webservice \
```

Comment:
Is this not also required in runmode?

Reply:
We didn't have it in runmode in sumo. I have an outstanding question about this in the [work item](https://ni.visualstudio.com/DevCentral/_workitems/edit/1856115).

Reply:
Updated to include in runmode also